<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>100</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter your token</string>
      </property>
     </widget>
     <widget class="QLabel" name="tokenOutput">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>220</y>
        <width>401</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="verifyButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>170</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Verify</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="tokenText">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>130</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Enter your shorthand token</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="selectDirectoryButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>311</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Directory</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="directoryText">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>311</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="saveRadio">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>40</y>
        <width>99</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>save</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="extdataRadio">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>40</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>extdata</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>Make sure this is correct</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="gameIDText">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>250</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Ideally, keep this the same per game. So like if you're uploading another Pokemon Y save, don't change the name from the previous save. Alright?</string>
      </property>
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::NoWrap</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="uploadButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Upload Save</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>311</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>Enter an ID/name for this game</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveGameIDButton">
      <property name="geometry">
       <rect>
        <x>284</x>
        <y>236</y>
        <width>56</width>
        <height>65</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="selectPreviousGameIDsButton">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>270</y>
        <width>252</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="text">
       <string>Select from previous ID/names</string>
      </property>
     </widget>
     <widget class="QPushButton" name="fetchButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>80</y>
        <width>311</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Fetch Saves</string>
      </property>
     </widget>
     <widget class="QPushButton" name="downloadButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>310</y>
        <width>311</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Download Save</string>
      </property>
     </widget>
     <widget class="QComboBox" name="downloadGameIDComboBox">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>240</y>
        <width>301</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>test option 1</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>220</y>
        <width>311</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>Game ID</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
